src/fields/hooks/beforeChange/promise.ts

```diff
@@ -3,7 +3,6 @@ import merge from 'deepmerge';
 import { Field, fieldAffectsData, TabAsField, tabHasName } from '../../config/types';
 import { Operation } from '../../../types';
 import { PayloadRequest, RequestContext } from '../../../express/types';
-import getValueWithDefault from '../../getDefaultValue';
 import { traverseFields } from './traverseFields';
 import { getExistingRowDoc } from './getExistingRowDoc';
 import { cloneDataFromOriginalDoc } from './cloneDataFromOriginalDoc';
```

Comment:
Looks like this function is no longer used in this file right?

Reply:
@JarrodMFlesch Good catch, just removed.